src/Symfony/Component/Mailer/Command/MailerDebugCommand.php

```diff
@@ -0,0 +1,129 @@
+ */
+class MailerDebugCommand extends Command
+{
+    /** {@inheritdoc} */
+    protected static $defaultName = 'debug:mailer:email:send';
+
+    /** @var MailerInterface */
+    private $mailer;
+
+    /** @var SymfonyStyle */
+    private $io;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(MailerInterface $mailer)
+    {
+        parent::__construct();
+
+        $this->mailer = $mailer;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setDescription('Send simple email message')
+            ->setDefinition([
+                new InputOption('from', null, InputOption::VALUE_REQUIRED, 'The from email of the message'),
+                new InputOption('to', null, InputOption::VALUE_REQUIRED, 'The to email of the message'),
+                new InputOption('subject', null, InputOption::VALUE_REQUIRED, 'The subject of the message'),
+                new InputOption('body', null, InputOption::VALUE_REQUIRED, 'The body of the message'),
+            ])
+            ->setHelp(
+                <<<EOF
+The <info>%command.name%</info> command creates and sends a simple email message.
+Usage:
+- <info>php %command.full_name%</info> will trigger the interactive mode
+- <info>php %command.full_name% --from=a@symfony.com --to=b@symfony.com --subject=Test --body=body</info>
+EOF
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->io = new SymfonyStyle($input, $output);
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws TransportExceptionInterface
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $email = $this->createEmail($input);
+
+        try {
+            $this->mailer->send($email);
+        } catch (TransportExceptionInterface $e) {
+            $this->io->error('Fail to send email.');
+            throw $e;
+        }
+
+        $this->io->success(
+            sprintf(
+                'Email was successfully sent to "%s".',
+                (string) $input->getOption('to')
+            )
+        );
+
+        return Command::SUCCESS;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        foreach ($input->getOptions() as $option => $value) {
+            if (null === $value) {
+                $input->setOption($option, $this->io->ask(sprintf('%s', ucfirst($option))));
+            }
+        }
+    }
+
+    private function createEmail(InputInterface $input): Email
+    {
+        return (new Email())
+            ->from($input->getOption('from'))
+            ->to($input->getOption('to'))
+            ->priority(Email::PRIORITY_HIGH)
+            ->subject($input->getOption('subject'))
+            ->text($input->getOption('body'))
+            ->html($input->getOption('body'));
```

Comment:
passing the same string to `text()` and `html()` does not make sense. It cannot be HTML and plain text at the same time.

Reply:
I might have read the doc a little too fast. I thought it was to allow controlling the 
- Content-Type: text/html;
- Content-Type: text/plain;
So you still receive the `text/plain` if your mail client doesn't support `text/html`
See https://symfony.com/doc/current/mailer.html#creating-sending-messages

But indeed it makes no sense to have both the same.

What would you propose ?

Reply:
Maybe wrap the text inside html/body/p ?

Reply:
Done :+1: 
Template stolen from https://github.com/leemunroe/responsive-html-email-template